Allow configuring MDC key names for trace_id, span_id, trace_flags (#11329)
open-telemetry/opentelemetry-java-instrumentation: instrumentation/log4j/log4j-context-data/log4j-context-data-2.17/library-autoconfigure/README.md

- `span_id`
- `trace_flags`

> If you want to customize the names of these keys, you can set system property or environment variable:

Reviewer: ```suggestion
These keys can be customized using the following system properties or environment variables:
```
